src/App.css

```diff
@@ -1,38 +1,206 @@
+  display: flex;
+  /* position: fixed;
+  left: 50%;
+  transform: translateX(-50%); */
+  z-index: -1;
+  flex-wrap: wrap;
+}
+
+.noteZone ul {
   display: flex;
   flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  flex-wrap: wrap;
+  margin: 0 0;
+  padding: 0;
+  height: 80vh;
+}
+
+/* Header */
+header {
+  display: flex;
+  height: 10vh;
+  margin: 30px 0 0 -10px;
+}
+
+/* Nav */
+/* Info input form*/
+.infoVisible {
+  position: fixed;
+}
+
+.infoInvisible {
+  display: none;
+}
+
+
+/* H1 */
+h1 {
+  font-size: 1.5rem;
+  margin-left: 150px;
+  position: fixed;
+}
+
+/* Input Form */
+.formVisible {
+  display: flex;
+  position: fixed;
+  margin: 60px 0 0 10px;
+}
+
+.formInvisible {
+  display:none;
+}
+
+.inputBox, .inputBoxDisabled {
+  width: 200px;
+  margin: 1px 0 0 10px;
+  padding: 10px;
+  border: 1px solid #4e57ef;
+  margin-left: 10px;
+}
+
+.inputBoxDisabled {
+  /* background-color: #4e57ef; */
+  border: 1px dashed #cfd7ff;
+  background-color: #ffff;
+  cursor: not-allowed;
+}
+
+/* Buttons Styling */
+.info, .input {
+  padding: 10px;
+  width: 40px;
+  position: fixed;
 }
 
-.App-link {
-  color: #61dafb;
+.info {
+  margin-left: 80px;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.countButton {
+  margin: 0 0 0 1000px;
+  padding: 0 5px 20px 5px;
+  position: fixed;
+  box-shadow: none;
+  font-size: 5rem;
+  box-shadow: 1px 1.25px 0px 0px #4e57ef;
+  line-height: 80px;
 }
+
+.speechVisible {
+  display: block;
+  position: fixed;
+  margin: 0 0 0 870px;
+  color: #cfd7ff;
+  transform:scale(14, 8);
+}
+
+.speechInvisible {
+  display: none;
+}
+
+.counterVisible {
+  display: block;
+  font-size: .9rem;
+  position: fixed;
+  margin: 20px 0 0 780px;
+  z-index: 1; 
+}
+
+.counterInvisible {
+  display: none;
+}
+
+
+.submitButton {
+  display: block;
+  position: fixed;
+  margin-left: 430px;
+  margin-top: .5px;
+  padding: 9.5px 11.5px 9.5px 11.5px;
+}
+
+.submitButtonDisabled {
+  display: none;
+}
+
+
+/* Main */
+/* Sticky Notes */
+.note {
+  border: 1px solid #4e57ef;
+  margin: 10px;
+  background-color: #cfd7ff;
+  box-shadow: 3px 4px 0px 0px #4e57ef;
+  width: 200px;
+  height: 80px;
+  padding: 10px;
+  overflow-wrap: break-word;
+}
+
+button {
+  border: 1px solid #4e57ef;
+  background-color: #ffff;
+  box-shadow: 1px 1.25px 0px 0px #4e57ef;
+  color: #4e57ef;
+  margin: 10px;
+}
+
+button:hover, button:focus {
+  border: 1px dotted #ffff;
+  background-color: #4e57ef;
+  box-shadow: 1px 1.25px 0px 0px #4e57ef;
+  color: #ffff;
+}
+
+button:active {
+  background-color: #cfd7ff;
+  box-shadow: inset -2px -2px #cfd7ff;
+}
+
+footer {
+  padding-block: 20px;
+}
+
+.xClose {
+  padding: 0px 6px 2px 6px;
+}
+
+  /* background-image: url("../assets/xp.png");
+  background-attachment: fixed;
+  background-size: cover;
+  background-color: #ffff; */
```

Comment:
maybe just do one media query @ 600px or so, adjusting the width/size of the noteZone/ individual notes?